src/main/resources/migrations/V8__create_table_tasks.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE tasks (
+    id SERIAL PRIMARY KEY,
+    server_id INT NOT NULL,
+	description VARCHAR NOT NULL,
+	due_date DATE NOT NULL,
+    due_time TIME NOT NULL,
+
+	CONSTRAINT fk_server
+    FOREIGN KEY(server_id)
+    REFERENCES servers(id)
```

Comment:
L'indentation est un peu cheloue non ? Pas très important en vrai mais bon, tant que j'y suis..!

Reply:
C'est juste, j'ai ouvert ça rapidement pour rajouter les `NOT NULL`, ça m'a échappé.